docs/sorting-algorithms/insertion-sort.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Insertion Sort
+slug: /sorting-algorithm/insertion-sort
+---
+
+## Introduction
+
+A simple sorting algorithm which works on the idea of picking elements from the unsorted part one by one and inserting them into their right positions in the sorted part. The array is virtually divided into 2 parts, a sorted one and an unsorted one. Just like we arrange a group of shuffled pages according to their page numbers, values form unsorted part are picked and placed at the right positions in the sorted portion.
+
+![Imgur](https://i.imgur.com/Ora9Q1J.png)
```

Comment:
Has the image been made by you?

Reply:
Yes I made it on Whimsical and uploaded its screenshot on Imgur - https://imgur.com/Ora9Q1J.png